exercises/diamond/examples/success-standard/src/Diamond.hs

```diff
@@ -0,0 +1,13 @@
+module Diamond (diamond) where
+
+import Data.Char (ord, chr)
+
+diamondH :: (Int -> String) -> (Char, (Int, Int)) -> String
+diamondH rb (c, (0, y)) = rb y ++ [c] ++ rb y
+diamondH rb (c, (x, y)) = rb y ++ [c] ++ rb x ++ [c] ++ rb y
+
+diamond :: Char -> [String]
+diamond = (\x -> x ++ tail (reverse x)) . f . abs . (64-) . ord
```

Comment:
the `abs` seems clumsy here, since you want to subtract 64 from result of ord. Do you prefer `(subtract 64) . ord` or `abs . (64-) . ord` ? I think the former, but unsure.